CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 13.1.0
+
+* Use `individu` `revenus_locatifs` if any to compute `f4ba` (revenus fonciers imposables)
+
+This may change any variable related to impot sur le revenu if `revenus_locatifs` is not zero and f4ba is not initialized.
```

Comment:
“change” as in “fix / correct / improve” or as in “raise an error”? 🤔
If it's the second, then it's a major bump.